server/meeting.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type Meeting struct {
+	ChannelId     string       `json:"channelId"`
+	Schedule      time.Weekday `json:"schedule"`
+	HashtagFormat string       `json:"hashtagFormat"` //Default: Jan02
+}
+
+func (p *Plugin) GetMeeting(channelId string) (*Meeting, error) {
+
+	meettingBytes, appErr := p.API.KVGet(channelId)
+	if appErr != nil {
+		return nil, appErr
+	}
+
+	var meeting *Meeting
+	if meettingBytes != nil {
+		if err := json.Unmarshal(meettingBytes, &meeting); err != nil {
+			return nil, err
+		}
+	} else {
+		//Return a default value
+		meeting = &Meeting{
+			Schedule:      time.Thursday,
+			HashtagFormat: "Jan02",
```

Comment:
0/5 maybe add (last dozen characters?) from the channel name? Not sure if the use of the default is typical.

Reply:
I will create a ticket to add this as a future enhancement.